execute/aggregate.go

```diff
@@ -21,7 +21,7 @@ type aggregateTransformation struct {
 
 type AggregateConfig struct {
 	plan.DefaultCost
-	Columns []string `json:"column"`
```

Comment:
We can't do this because it makes it so `aggregateWindow` won't work correctly.

Reply:
I'm missing something.  This changes the serialization format, but not much else, right?  All the go tests passed, are we missing coverage?

Reply:
Ah, I think I misunderstood the aggregate config change. It seems it has a special case for having both `column` and `columns`.